lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -157,6 +163,7 @@ def partition_pdf(
         output_format: Optional[str] = None,
         text_extraction_options: dict[str, Any] = {},
         source: str = "",
+        promote_title: bool = False,
```

Comment:
Can you check this from the `text_extraction_options`? You'll also need to add documentation in the top level partition call to say that `promote_title` is an accepted argument in `text_extraction_options` and explain its functionality.

Reply:
done

Reply:
adding a 'output_label_options' option for driving all output label related functionalities.